snmp/tests/test_profiles.py

```diff
@@ -60,6 +60,67 @@ def run_profile_check(recording_name):
     check.check(instance)
 
 
+def test_cisco_voice(aggregator):
+    run_profile_check('cisco_icm')
+
+    tags = ['snmp_profile:cisco_icm', 'snmp_host:test'] + common.CHECK_TAGS
+
+    resources = ["hrSWRunPerfMem", "hrSWRunPerfCPU", "hrSWRunStatus"]
+
+    for resource in resources:
+        aggregator.assert_metric('snmp.{}'.format(resource), metric_type=aggregator.GAUGE, tags=tags)
+
+    cvp_gauges = [
+        "ccvpSipIntAvgLatency1",
+        "ccvpSipIntAvgLatency2",
+        "ccvpSipIntConnectsRcv",
+        "ccvpSipIntNewCalls",
+        "ccvpSipRtActiveCalls",
+        "ccvpSipRtTotalCallLegs",
+        "ccvpLicRtPortsInUse",
+    ]
+
+    cvp_counts = [
+        "ccvpLicAggMaxPortsInUse",
+    ]
+    for cvp in cvp_gauges:
+        aggregator.assert_metric('snmp.{}'.format(cvp), metric_type=aggregator.GAUGE, tags=tags)
+
+    for cvp in cvp_counts:
+        aggregator.assert_metric('snmp.{}'.format(cvp), metric_type=aggregator.RATE, tags=tags)
+
+    ccms = ["ccmRegisteredGateways", "ccmRejectedPhones", "ccmRegisteredPhones", "ccmUnregisteredPhones"]
+
+    for ccm in ccms:
+        aggregator.assert_metric('snmp.{}'.format(ccm), metric_type=aggregator.RATE, tags=tags)
+
+    calls = [
+        "cvCallVolPeerIncomingCalls",
+        "cvCallVolPeerOutgoingCalls",
+        "cvCallVolMediaIncomingCalls",
+        "cvCallVolMediaOutgoingCalls",
+    ]
+
+    for call in calls:
+        aggregator.assert_metric('snmp.{}'.format(call), metric_type=aggregator.GAUGE, tags=tags)
+
+    dial_controls = [
+        "dialCtlPeerStatsAcceptCalls",
+        "dialCtlPeerStatsFailCalls",
+        "dialCtlPeerStatsRefuseCalls",
+        "dialCtlPeerStatsSuccessCalls",
+    ]
```

Comment:
I think I missed this in the last review, but these are `AbsoluteCounter32` type. I'm not sure if the snmp check supports that as a counter type and if we should consider adding it. cc @florimondmanca 

https://github.com/DataDog/integrations-core/blob/50baa2606bdaf8e059a60346e037d07c534f4aaf/snmp/datadog_checks/snmp/pysnmp_inspect.py#L34-L39

Reply:
Hey,

By default metrics that are not in the explicit list of SNMP class names you linked to will be submitted as a `gauge`. If that’s what we want here (assuming from the tests it looks like it is) then fine, otherwise if we'd like this to be submitted as a `rate` then we need to update the lists and possibly the function that does the inference (in `metrics.py`).

Reply:
After a conversation, we decided to use the forced-type option to make this a monotonic count, since these metrics are the only ones we've seen of this type so far. If we encounter more, then we can re-evaluate and add it to the known types.